src/components/subscriptions/LicenseStatus.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export default function LicenseStatus({ user }) {
+  const licenseStatus = useMemo(
+    () => {
+      switch (user.licenseStatus) {
+        case 'active':
```

Comment:
Status stuff still seems up in the air, but what you have so far around statuses seems like a good start/placeholder